samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -0,0 +1,14 @@
+.. _peripheral_fast_pair:
+
+Bluetooth: Fast Pair
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Fast Pair sample demonstrates how to use the Fast Pair in the |NCS|.
+
+.. note::
+   The feature is still under development.
+   Documentation will be introduced soon.
```

Comment:
Instead of this note, we need at least the following sections with basic information:

- Requirements (supported DKs)
- Overview (general description of sample flow)
- Building and running (how to build)
- Testing (how to test; you can put some very basic info there).

See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html for reference.

Reply:
Will be added in a separate PR. I will open the mentioned PR as soon as this one goes in.